Menu/Provider/GroupMenuProvider.php

```diff
@@ -80,6 +88,12 @@ public function get($name, array $options = array())
                         'admin' => $admin,
                     );
                 } else {
+                    if (!empty($item['roles']) && !$this->checker->isGranted($item['roles'])
```

Comment:
here there is a `((case and case) or case)` statement. That's why the first two are on one line.